calico_node/tests/st/bgp/peer.py

```diff
@@ -34,12 +33,5 @@ def create_bgp_peer(host, scope, ip, asNum, metadata=None):
     if metadata is not None:
         testdata['metadata'] = metadata
 
-    host.writefile("testfile.yaml", yaml.dump(testdata))
-    host.calicoctl("create -f testfile.yaml")
-
-def clear_bgp_peers(host):
```

Comment:
I'd like to understand why we no longer want to do this.  IIUC there are existing tests that include this; I think they are probably all skipped, at the moment, but how do we know that they won't need this step when they are reenabled again?

Reply:
Ah, that's fine - didn't realize that it was something you added during our efforts to get things working.